src/main/java/com/shanebeestudios/skbee/elements/other/conditions/CondIsLocked.java

```diff
@@ -0,0 +1,40 @@
+package com.shanebeestudios.skbee.elements.other.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.block.Beacon;
+import org.bukkit.block.Block;
+import org.bukkit.block.Container;
+
+@Name("Is Locked")
+@Description("Check if a container or beacon is in a locked state")
```

Comment:
you and you damn periods
```suggestion
@Description("Check if a container or beacon is in a locked state.")
```